build.edn

```diff
@@ -0,0 +1,8 @@
+{:lib toyokumo/ring-middleware-csp
+ :version "0.4.{{git/commit-count}}"
```

Comment:
It is recommended to specify license, because clojars requires `license` in pom.xml for new projects.
https://github.com/clojars/clojars-web/pull/874

e.g. https://github.com/liquidz/build.edn/blob/ab331f5c651dacccc906e2d41cedcbe6b3e6932c/build.edn#L4-L5